### ISSUE TITLE
Fix CI builds for stable Numpy and Astropy

### ISSUE THREAD

**Issue by cdeil** [hipspy/hips#6]
@bsipocz - I set up this new package `hips` using the package-template cookie-cutter.

One error I'm gettting is this:
```
PackageNotFoundError: Package not found: '' Packages missing in current linux-64 channels: 
  - numpy stable*
  - astropy stable*

You can search for packages on anaconda.org with

    anaconda search -t conda numpy

(and similarly for the other packages)

You may need to install the anaconda-client command line client with

    conda install anaconda-client

Note that you have pinned specs in /home/travis/miniconda/envs/test/conda-meta/pinned:

    ['numpy stable*', 'astropy stable*', 'healpy', 'reproject', 'matplotlib', 'sphinx  1.5.6*']
```
https://travis-ci.org/hipspy/hips/jobs/236899940#L483

Did I mess up the `.travis.yml` setup somehow?
Or is this a bug in the package-template?
Do you know what the fix is or should I investigate?

**Comment by bsipocz** [comment]
This is something weird, I'll try to have a look.

**Comment by bsipocz** [comment]
I have a branch with a cleaned up travis and appveyor file, I'll open a PR once everything is fixed upstream.
First of all, neither ``astropy`` nor ``numpy`` should be listed in ``CONDA_DEPENDENCIES``. It's a known issue in ci-helpers (https://github.com/astropy/ci-helpers/issues/84), and at some point it may be get fixed. 

The other issues is about changes I added to ci-helpers today (when a package is not available on conda we try to install it with pip. there are still some bugs with a few corner cases you hit there, so I'm trying to iron them out before opening a PR here).

**Comment by cdeil** [comment]
@bsipocz - You're https://www.youtube.com/watch?v=mNU3aIJs88g . Thanks!

**Comment by cdeil** [comment]
This issue was fixed by @bsipocz in #9. Closing now.
